pom.xml

```diff
@@ -32,7 +32,7 @@
     <skipIllumina>true</skipIllumina>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <jackson.version>2.9.9</jackson.version>
+    <jackson.version>2.9.9.2</jackson.version>
```

Comment:
Will we be affected by this? https://github.com/FasterXML/jackson-databind/issues/2395

Reply:
Probably. I will rollback.